README.md

```diff
@@ -139,6 +153,99 @@ You can find an example of the SQL file at `tests/sql_script.sql`.
 
 Default: `null`
 
+Type: `str`
+
+### `mssql_enable_sql_agent`
+
+Set this variable to `true` or `false` to enable or disable the SQL agent.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_install_fts`
+
+Set this variable to `true` or `false` to install or remove the
+`mssql-server-fts` package that provides full-text search.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_enable_ha`
+
+Set this variable to `true` or `false` to install or remove the
+`mssql-server-ha` package and enable or disable the `hadrenabled` setting.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_tune_for_fua_storage`
+
+Set this variable to `true` or `false` to enable or disable settings that
+improve performance on hosts that support Forced Unit Access (FUA) capability.
+
+Only set this variable to `true` if your hosts are configured for FUA
+capability.
+
+When set to `true`, the role applies the following settings:
+
+* Set the `traceflag 3979 on` setting to enable trace flag 3979 as a startup
+parameter
+* Set the `control.alternatewritethrough` setting to `0`
```

Comment:
do you mean `1` here?

Reply:
According to MS docs it should be zero.
https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-performance-best-practices?view=sql-server-ver15#sql-server-and-forced-unit-access-fua-io-subsystem-capability
The same in Louis scripts: https://github.com/redhat-cop/sqlserver-coi/blob/master/scripts/mssql-install-demo/enterprise-storage.sh

Reply:
So it is the same as the default value?

Reply:
That is - the default value is `0` - setting `mssql_tune_for_fua_storage: true` will set it to `0` - so why set `control.alternatewritethrough` at all?  Can it be set to any value other than `0`?

Reply:
Even though `0` is the default, users might have it set to `1`, for some internal reasons. In this case, the role must set it back to `0`.
I do not really know how to handle disabling this `mssql_tune_for_fua_storage`. For now, I just wrote what the role does explicitly to be fair.
Indeed the role always sets this setting to `0`. Another way would be to usnet this setting. Unsetting means setting the default value - again `0`. I checked that the default for this setting is `0` with Amit. Explicitly setting to `0` is more clear than unsetting then, and produces the same result. Do you have an idea how to improve it?